wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -708,6 +708,28 @@ def admin_theme_classname(context):
     return f"w-theme-{theme_name} w-density-{density_name} w-contrast-{contrast_name}"
 
 
+@register.simple_tag(takes_context=True)
+def admin_theme_color_scheme(context):
+    """
+    Retrives the color-scheme of the active theme for current user.
+    """
+    user = context["request"].user
+    # print('userdwa',user)
+    # user.wagtail_userprofile.theme = "light"
+    theme_name = (
+        user.wagtail_userprofile.theme
+        if hasattr(user, "wagtail_userprofile")
```

Comment:
```suggestion
        getattr(user, "wagtail_userprofile", None)
```

We should be able to streamline this, try this out, removing the else block. Then below we would reverse the check, checking for dark first. Meaning we don't need to declare light twice.